code/test/util.js

```diff
@@ -455,23 +690,23 @@ const getState = async (wallet) => {
 
 // check OneWallet state
 const assertStateEqual = async (expectedState, actualState, checkNonce = false) => {
-  assert.deepEqual(actualState.identificationKey, expectedState.identificationKey, 'wallet.identificationKey is incorrect')
-  assert.deepEqual(actualState.identificationKeys, expectedState.identificationKeys, 'wallet.identificationKeys is incorrect')
-  assert.deepEqual(actualState.forwardAddress, expectedState.forwardAddress, 'wallet.forwardAddress is incorrect')
-  assert.deepEqual(actualState.info, expectedState.info, 'wallet.info is incorrect')
-  assert.deepEqual(actualState.oldInfo, expectedState.oldInfo, 'wallet.oldInfos is incorrect')
-  assert.deepEqual(actualState.innerCores, expectedState.innerCores, 'wallet.innerCores is incorrect')
-  assert.deepEqual(actualState.rootKey, expectedState.rootKey, 'wallet.rootKey is incorrect')
-  assert.deepEqual(actualState.version, expectedState.version, 'wallet.version is incorrect')
-  assert.deepEqual(actualState.spendingState, expectedState.spendingState, 'wallet.spendingState is incorrect')
+  assert.deepEqual(expectedState.identificationKey, actualState.identificationKey, 'wallet.identificationKey is incorrect')
```

Comment:
actual state is at the first argument https://www.chaijs.com/api/assert/

Reply:
Updating 🙏